src/languages/en.ts

```diff
@@ -3260,6 +3261,9 @@ export default {
             addedWithPrimary: 'Some members were added with their primary logins.',
             invitedBySecondaryLogin: ({secondaryLogin}) => `Added by secondary login ${secondaryLogin}.`,
             membersListTitle: 'Directory of all workspace members.',
+            importMembers: 'Import members',
+            importedMembersMessage: (columnCounts: number) =>
+                `We found *${columnCounts} columns* in your spreadsheet. Select *Email* next to the column that contains emails. You can also select *Role* next to the column that sets users roles.`,
```

Comment:
@Guccio163 One last thing from me, was this text agreed somewhere? I don't remember where we discussed texts.

If so, I think everything that is between `*` should actually yield a bold text
<img width="370" alt="image" src="https://github.com/user-attachments/assets/e70d44de-d770-4e08-8655-fd25568280ca">

Reply:
I can't be sure about that because I took that task from @filip-solecki with import functionality already made (and hence translations). To be fair I cannot find any conversation touching this topic so can you please assign someone from translation team to check these? I'll also check how this text should look like from doc, hopefully `*` will be explained.

Reply:
Nice, coming from [this thread](https://expensify.slack.com/archives/C06ML6X0W9L/p1726603502772629?thread_ts=1726494652.467189&cid=C06ML6X0W9L), let's use the same message we use for categories.

<img width="374" alt="image" src="https://github.com/user-attachments/assets/6e450a28-06be-441a-a6a2-2a33e4f868a3">

Reply:
@rlinoz Changes applied, it turns out that it was fixed in the whole `ImportSpreadsheetColumns' component in `Tags` PR mentioned above, which standardises header in all import sections 👌

Reply:
@Guccio163 I guess we can remove this line then right? I don't see it being used anywhere else.

Reply:
Oh yes, I knew I forgot something!

Reply:
@rlinoz fixed 👍